jupyter-book/preprocessing_visualization/preprocessing.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>Of course it is not perfectly answered yet what the complete style of the book is going to be (we're talking next week about this), but I would be happy to see a more elaborate and biologically motivated introduction here. For example some story where bad quality data could lead to wrong conclusions and that it is very difficult to get to any conclusions without nice visualizations in the first place etc.</p><p> </p><p>Afterwards, mention what the chapter is about + where the data comes from etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>